components/more-less/more-less.js

```diff
@@ -231,7 +239,8 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 
 	__expand() {
 		this.__transitionAdded = true;
-		this.__contentHeight = `${this.__content.scrollHeight}px`;
+		this.__contentHeight = 'unset';
```

Comment:
While it seems to be working fine while I test it, there seems to be some inconsistency with when `__contentHeight` and `__maxHeight` are set. On initialization `__maxHeight` is unset. It would probably be a good idea to initialize that to something. 

After expanding or shrinking `__contentHeight` is unset and never re-set. Does the component still work as expected if you set `__maxHeight` and remove `__contentHeight` entirely?

Reply:
To correct my previous comment, I think there is actually a use for `__contentHeight`. The height attribute is used for the animation [here](https://github.com/BrightspaceUI/core/blob/fb51e8b8be04af51722e2e97cabc923b5f97fea4/components/more-less/more-less.js#L65). Changing these to `unset` is likely why the animation has stopped working.

I think what would fix this is to set the height with 2 separate cases:
1. When in shrunken form, set `this.__contentHeight` to the smallest of  `${this.__content.scrollHeight}` and  `this.height`
2. When in expanded form: `this.__contentHeight = ${this.__content.scrollHeight}px`

The problem with that though is it might not be simple to compare the heights in case 1 since the `height` property is a string 🤔

Reply:
After thinking about this for a bit, I think there might be an easier way. It should be possible to get the animations working properly by changing the transition animation to use `max-height` instead of `height`. Then the two cases are easier to code:
1. When in shrunken form set: `this.maxHeight = this.height`
2. When in expanded form set: `this.maxHeight = ${this.__content.scrollHeight}px`

And then I think you might be able to remove the `__contentHeight` property as it would be unused.

Reply:
Change has been made, thanks!